web/src/unread.ts

```diff
@@ -418,6 +431,7 @@ class UnreadTopicCounter {
 
             if (user_topics.is_topic_unmuted_or_followed(stream_id, topic)) {
                 unmuted_count += topic_count;
+                msg_ids = [...msg_ids, ...msgs];
```

Comment:
These variable names are very confusing; `msgs` if they're both lists of message IDs, given them longer names of the form `foo_message_ids`.